### PR TITLE
do not use jinja in assert, we can use filters directly

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: tests/test_playbooks/host_info.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@
     - name: check host details
       assert:
         that:
-          - host_info['host']['name'] == "test-host.{{ host.domain }}"
+          - host_info['host']['name'] == "test-host.{}".format(host.domain)
```
Comment (mdellweg) [inline tests/test_playbooks/host_info.yml:57]: As if that were easier...
So it looks like the whole condition is put through jinja2. Is that correct?

**Review by mdellweg — APPROVED** [review]

File: tests/test_playbooks/host_info.yml (hunk 1 above)
Comment (evgeni) [reply tests/test_playbooks/host_info.yml:57]: there is no jinja here anymore, just plain python :D

(no, I have not the slightes idea why this should be any better)

Comment (evgeni) [reply tests/test_playbooks/host_info.yml:57]: OTOH, you can use `|filter`, so maybe there is still some jinja... the hell I know

Comment (mdellweg) [reply tests/test_playbooks/host_info.yml:57]: That's what it looks to me. Whatever you assert here seems to be processed by jinja.

Comment (evgeni) [reply tests/test_playbooks/host_info.yml:57]: instead of melting my brain over this, I'll go get some sushi :)

**Pull request merged**